packages/core/src/awsService/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -715,9 +715,9 @@ export class IamPolicyChecksWebview extends VueWebview {
     }
 
     public pushCustomCheckDiagnostic(diagnostics: vscode.Diagnostic[], finding: any, isBlocking: boolean) {
-        const message = `${finding.findingType}: ${finding.message} - Resource name: ${finding.resourceName}, Policy name: ${finding.policyName}`
+        let message = `${finding.findingType}: ${finding.message} - Resource name: ${finding.resourceName}, Policy name: ${finding.policyName}`
         if (message.includes('existingPolicyDocument')) {
-            message.replace('existingPolicyDocument', 'reference document')
+            message = message.replace('existingPolicyDocument', 'reference document')
```

Comment:
this could remain `const` by using a ternary (which has readability benefits as well)

    const message = finding.findingType.includes('existingPolicyDocument') ? ... : ...

Reply:
Will update it, thanks!